cypress.config.js

```diff
@@ -10,10 +10,17 @@ module.exports = defineConfig({
           const email = faker.internet.email();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
           };
+        },
+        generateArticle() {
+          return {
+            title: faker.word.verb(),
```

Comment:
It's better to use `faker.lorem.words(1)` instead of `faker.word.verb()` for the title. This way, you ensure that the title is a single word, as `faker.word.verb()` can return a verb phrase.